libraries/abstractions/common_io/include/iot_sdio.h

```diff
@@ -0,0 +1,709 @@
+/*
+ * Amazon FreeRTOS Common IO V1.0.0
+ * Copyright (C) 2019 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+
+
+/**
+ * @file    iot_sdio.h
+ * @brief   This file contains all the SDIO HAL API definitions.
+ *          Here we define SDIO driver as a protocol layer driver which builds on top of SDMMC
+ *          host driver. SDMMC host driver operates at physical layer and provide APIs to send
+ *          commands to the slave device and receive command responses, send and receive data,
+ *          and handle error conditions on the bus.
+ *
+ *          SD/SDIO/MMC driver can be built on top of SDMMC host. This interface only define
+ *          the APIs for SDIO applications, the interface can be expanded to support SD/MMC either
+ *          by expanding the APIs in this file or adding separate files for SD/MMC
```

Comment:
Can you please add a period to the end of the sentence?

Reply:
added.